flink-formats/flink-csv/src/main/java/org/apache/flink/formats/csv/CsvRowDataDeserializationSchema.java

```diff
@@ -81,19 +82,34 @@ private CsvRowDataDeserializationSchema(
     @Internal
     public static class Builder {
 
-        private final RowType rowType;
+        private final RowType rowResultType;
         private final TypeInformation<RowData> resultTypeInfo;
         private CsvSchema csvSchema;
         private boolean ignoreParseErrors;
 
+        /**
+         * Creates a CSV deserialization schema for the given {@link TypeInformation} with optional
+         * parameters.
+         */
+        public Builder(
+                RowType rowReadType,
+                RowType rowResultType,
```

Comment:
The naming used here as `rowReadType` and `rowResultType` is not clear enough. Javadoc is required. Why not use `originalRowType` `projectedRowType`?

Reply:
Good point, the JavaDoc of the fields was originally not there so I missed it.
The naming comes from the original signature:
```
public Builder(RowType rowType, TypeInformation<RowData> resultTypeInfo) 
```
The ideas is to underline using which CsvSchema the data is going to be read from the file and what is the expected output of the result. Projection seems a bit a too specific term used in the SQL optimization world. I can see this being used outside of this scope (simple filtering for whatever other reasons or expansion of the nested fields according to a "wider" row), depending on the converter used.
I've added the the missing javadoc.